cpp/cmake/thirdparty/CUSPATIAL_GetCUDF.cmake

```diff
@@ -0,0 +1,45 @@
+#=============================================================================
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+
+function(find_and_configure_cudf VERSION)
+
+    if(TARGET cudf::cudf)
+        return()
+    endif()
+
+    cuspatial_save_if_enabled(BUILD_TESTS)
+    cuspatial_save_if_enabled(BUILD_BENCHMARKS)
+    CPMFindPackage(NAME        cudf
+        VERSION                ${VERSION}
+        GIT_REPOSITORY         https://github.com/rapidsai/cudf.git
+        GIT_TAG                branch-${VERSION}
+        GIT_SHALLOW            TRUE
+        SOURCE_SUBDIR          cpp
+        OPTIONS                "BUILD_TESTS OFF"
+                               "BUILD_BENCHMARKS OFF"
+        FIND_PACKAGE_ARGUMENTS "COMPONENTS testing")
+    cuspatial_restore_if_enabled(BUILD_TESTS)
+    cuspatial_restore_if_enabled(BUILD_BENCHMARKS)
+
+    # Make sure consumers of cuspatial can see cudf::cudf
+    fix_cmake_global_defaults(cudf::cudf)
+    # Make sure consumers of cuspatial can see cudf::cudftestutil
+    fix_cmake_global_defaults(cudf::cudftestutil)
+endfunction()
+
+set(CUSPATIAL_MIN_VERSION_cudf "${CUSPATIAL_VERSION_MAJOR}.${CUSPATIAL_VERSION_MINOR}")
```

Comment:
I think this doesn't match the CMake project name here: https://github.com/rapidsai/cuspatial/pull/374/files#diff-1bba462ab050e89360fd88110a689e85ee037749cea091a1848ab574381d3795R31 which causes them to be empty strings